### PR TITLE
feat: [vmwareengine] new module for vmwareengine

### PR DESCRIPTION
**Pull request by lqiu96**
New Client has been generated with this command:

```
python3.9 generation/new_client/new-client.py generate \
  --api_shortname=vmwareengine \
  --proto-path=google/cloud/vmwareengine \
  --name-pretty="Google Cloud VMware Engine" \
  --product-docs="https://cloud.google.com/vmware-engine/" \
  --api-description="Easily lift and shift your VMware-based applications to Google Cloud without changes to your apps, tools, or processes."
```

### CODE REVIEW

**Comment by lqiu96** [comment]
This doesn't look like the intended behavior. Not sure why it's updating the pom.xml for other modules.

File: java-accessapproval/google-cloud-accessapproval-bom/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-pom-parent</artifactId>
-    <version>1.1.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-java:current} -->
+    <version>1.1.0-SNAPSHOT</version><!-- {x-version-update:google-cloud-pom-parent:current} -->
```
Comment (suztomo) [inline java-accessapproval/google-cloud-accessapproval-bom/pom.xml:12]: @ddixit14  The script needs to be updated to use google-cloud-java tag.

Comment (ddixit14) [reply java-accessapproval/google-cloud-accessapproval-bom/pom.xml]: This PR shouldn't update this file. Right?

File: java-vmwareengine/.repo-metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "api_shortname": "vmwareengine",
+  "name_pretty": "Google Cloud VMware Engine",
+  "product_documentation": "https://cloud.google.com/vmware-engine/",
+  "api_description": "Easily lift and shift your VMware-based applications to Google Cloud without changes to your apps, tools, or processes.",
+  "client_documentation": "https://cloud.google.com/java/docs/reference/google-cloud-vmwareengine/latest/overview",
+  "release_level": "preview",
+  "transport": "grpc",
+  "language": "java",
+  "repo": "googleapis/java-vmwareengine",
```
Comment (lqiu96) [inline java-vmwareengine/.repo-metadata.json:10]: I think this should be `googleapis/google-cloud-java`

Comment (suztomo) [reply java-vmwareengine/.repo-metadata.json:10]: I feel this can be fixed later. Do you see this field in JSON used somewhere?

Comment (lqiu96) [reply java-vmwareengine/.repo-metadata.json:10]: Ok, I'll create an issue for this then.

File: java-vmwareengine/README.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
+# Google Google Cloud VMware Engine Client for Java
```
Comment (suztomo) [inline java-vmwareengine/README.md:1]: Typo

**Review by suztomo — CHANGES_REQUESTED** [review]

File: java-vmwareengine/README.md (hunk 3 above)
Comment (lqiu96) [reply java-vmwareengine/README.md:1]: I coped the `name_pretty` value in DRIFT: 
image

Comment (lqiu96) [reply java-vmwareengine/README.md:1]: I copied the `display name` value on drift:
image

Should I change it to `Cloud VMware Engine`?

Comment (suztomo) [reply java-vmwareengine/README.md:1]: Can you change this line "Google Cloud VMware Engine Client for Java"? No need to re-run the script.

Comment (lqiu96) [reply java-vmwareengine/README.md:1]: Wouldn't it get overwritten the next time owlbot post processor runs? I thought the README template is from: https://github.com/googleapis/synthtool/blob/934c51d74f1ec59a20892525248de1bed8fec638/synthtool/gcp/templates/java_library/README.md?plain=1#L5

Comment (lqiu96) [reply java-vmwareengine/README.md]: Ok, post processor doesn't overwrite this. I assumed the post-processor created the README via synthtool: https://github.com/googleapis/google-cloud-java/blob/0f71f527a7e5c9812767b3c9e0820bc033b57510/generation/new_client/new-client.py#LL223C12-L223C12

If not, how was the README.md created?

**Review by suztomo — APPROVED** [review]

**Pull request merged**